opencensus/stats/measurement_map.py

```diff
@@ -51,10 +60,16 @@ def attachments(self):
 
     def measure_int_put(self, measure, value):
         """associates the measure of type Int with the given value"""
+        if value < 0:
+            # Should be an error in a later release.
+            logger.warning("Cannot record negative values")
```

Comment:
Note that we're breaking with the java impl here in that we don't mark the map invalid until record time.